### DISCUSSION TITLE
How to get   fill: rgb(252, 222, 77) value from a style

### DISCUSSION THREAD

**Discussion by narendarbhogala** [seleniumbase/SeleniumBase#2364] [General]
How to get   fill: rgb(252, 222, 77) value from a style attribute

table.css-hg9onk:nth-child(3) > tbody:nth-child(2) > tr:nth-child(1) > td:nth-child(4) > span:nth-child(1) > div:nth-child(1) > span:nth-child(1) > span:nth-child(1) > svg:nth-child(1) > g:nth-child(1) > path:nth-child(2)

**Comment by mdmintz** [comment]
You can use this method:

```python
self.get_attribute(selector, attribute)
```

There are some other similar methods:

```python
self.get_property(selector, property)

self.get_property_value(selector, property)
```
